RecoParticleFlow/PFClusterProducer/plugins/LegacyPFClusterProducer.cc

```diff
@@ -65,6 +70,7 @@ class LegacyPFClusterProducer : public edm::stream::EDProducer<> {
     desc.add<edm::InputTag>("PFRecHitsLabelIn");
     desc.add<edm::ESInputTag>("pfClusterParams");
     desc.add<edm::InputTag>("recHitsSource");
+    desc.add<bool>("usePFThresholdsFromDB", "True");
```

Comment:
shouldn't this be
```suggestion
    desc.add<bool>("usePFThresholdsFromDB", true);
```
?